templates/bot/ts/workflow/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "command-response-bot",
+    "version": "1.0.0",
+    "description": "Microsoft Teams Toolkit Command and Response Bot Sample",
+    "author": "Microsoft",
+    "license": "MIT",
+    "main": "./lib/index.js",
+    "scripts": {
+        "dev:teamsfx": "env-cmd --silent -f .env.teamsfx.local npm run dev",
+        "dev": "nodemon --watch ./src --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
+        "build": "tsc --build && shx cp -r ./src/adaptiveCards ./lib/src",
+        "start": "node ./lib/src/index.js",
+        "watch": "nodemon --watch ./src --exec \"npm run start\"",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com"
+    },
+    "dependencies": {
+        "@microsoft/adaptivecards-tools": "^1.0.0",
+        "@microsoft/teamsfx": "^1.0.0",
+        "botbuilder": "~4.16.0",
+        "restify": "^8.5.1"
+    },
+    "devDependencies": {
+        "@types/restify": "8.4.2",
+        "env-cmd": "^10.1.0",
+        "nodemon": "^2.0.7",
+        "shx": "^0.3.4",
+        "ts-node": "~9.1.1",
```

Comment:
#5663 updates ts, ts-node versions, tsconfig. Keep the consistency with those changes.